lms/templates/index.html

```diff
@@ -87,16 +87,18 @@
   </section>
 </section>
 
-<section id="video-modal" class="modal home-page-video-modal video-modal">
-  <div class="inner-wrapper">
-    <%
-      youtube_video_id = homepage_promo_video_youtube_id
-      if self.stanford_theme_enabled():
-        youtube_video_id = "2gmreZObCY4"
-    %>
-    <iframe title="YouTube Video" width="640" height="360" src="//www.youtube.com/embed/${youtube_video_id}?showinfo=0" frameborder="0" allowfullscreen></iframe>
-  </div>
-</section>
+% if show_homepage_promo_video:
+  <section id="video-modal" class="modal home-page-video-modal video-modal">
+    <div class="inner-wrapper">
+      <%
+        youtube_video_id = homepage_promo_video_youtube_id
```

Comment:
Assuming you remove the `stanford_theme_enabled` check below, this assignment becomes superfluous and could be removed by updating its reference.

At that point, this entire `<% ... %>` can be deleted!